planetiler-core/src/main/java/com/onthegomap/planetiler/geo/TileCoord.java

```diff
@@ -92,7 +92,7 @@ private static long hilbertIndexToXY(int n, int i) {
   }
 
   // Ignore warnings about nested scopes amd code duplication.
-  @SuppressWarnings({"java:S1199", "common-java:DuplicatedBlocksSource"})
+  @SuppressWarnings({"java:S1199", "common-java:DuplicatedBlocks"})
```

Comment:
this is not pretty but IMHO we should just leave the nested scope formatting to preserve the original code structure